spec/features/admin/orders_spec.rb

```diff
@@ -110,4 +110,54 @@
       expect(page).to have_current_path spree.edit_admin_order_path(incomplete_order)
     end
   end
+
+  context "save the filter params" do
+
+    scenario "when reloading the page" do
+      user = create(:user, email: 'an@email.com')
+      shipping_method = create(:shipping_method, name: "UPS Ground")
+      order = create( :order,
+                      distributor: distributor,
+                      order_cycle: order_cycle,
+                      user: user,
+                      number: "R123456",
+                      state: 'complete',
+                      payment_state: 'balance_due',
+                      completed_at: 1.day.ago)
```

Comment:
This way of formatting is brittle to change. If you rename `order` then you have to change the indent of all parameters. And if you add a parameter at the end, you need to change the current last parameter as well. This formatting is more open to change:
```suggestion
      order = create(
        :order,
        distributor: distributor,
        order_cycle: order_cycle,
        user: user,
        number: "R123456",
        state: 'complete',
        payment_state: 'balance_due',
        completed_at: 1.day.ago,
      )
```

Reply:
Yes, you're right, will squash it ;)